### ISSUE TITLE
`mouse_backward` and `mouse_forward` chatter not blocked

### ISSUE THREAD

**Issue by toiglak** [mcmonkeyprojects/KeyboardChatterBlocker#36]
Application correctly registers that chatter occurred (`chatter.wav` is played), but the second click is not blocked (chatter still occurs).

I tested this by setting `mouse_backward`'s chatter threshold to `1000` and double clicking that key. `chatter.wav` played, but applications that I tested this on (file explorer and chrome) received both clicks. The same happens with `mouse_forward`.

`mouse_left` works as expected.

**Comment by mcmonkey4eva** [comment]
Oh this was extremely confusing to diagnose.

I could see the detections, was confident it was blocking, but I had the same result as you - it was registering in programs anyway.

I searched a while for possible code mistakes, alternative mouse hooks that may be relevant, etc.

But then ... I held the key down, and noticed nothing moved - until I released it.
... Programs don't listen to the forward/backward button being pressed. They're listening for it being released! And KeyboardChatterBlocker allows duplicate releases through (to avoid bugs with keys accidentally getting virtually stuck down).

I updated the code to block key release for these two mouse buttons, and have published as [Release 1.16](https://github.com/mcmonkeyprojects/KeyboardChatterBlocker/releases).

Seems to be perfectly blocking it now for me!
